lib/msal-browser/src/interaction_client/PopupClient.ts

```diff
@@ -114,7 +114,7 @@ export class PopupClient extends StandardInteractionClient {
             const state = this.validateAndExtractStateFromHash(hash, InteractionType.Popup, validRequest.correlationId);
 
             // Remove throttle if it exists
-            ThrottlingUtils.removeThrottle(this.browserStorage, this.config.auth.clientId, authCodeRequest.authority, authCodeRequest.scopes);
+            ThrottlingUtils.removeThrottle(this.browserStorage, this.config.auth.clientId, authCodeRequest.authority, authCodeRequest.scopes, undefined, authCodeRequest);
```

Comment:
Can we remove `authCodeRequest.authority` and `authCodeRequest.scopes` from this if we're going to pass in the entire object anyway?

Reply:
Yes, we were trying to avoid adding a breaking change but I guess it makes sense since we have a major version release coming up. I'll try to update this today.

Reply:
Done.